server/app/views/components/ButtonStyles.java

```diff
@@ -0,0 +1,65 @@
+package views.components;
+
+import views.style.BaseStyles;
+import views.style.StyleUtils;
+
+/**
+ * A collection of styles for buttons throughout CiviForm. Importantly, the public style Strings in
+ * this file should be **generalizable**, meaning they are not designed for a particuar use case.
+ * Styles should not be called i.e. BUTTON_SUBMIT_APPLICATION for example. If a specific change is
+ * required for a use case, use a general style and add more classes in the client code.
+ */
+public final class ButtonStyles {
+  /**
+   * Base styles for buttons in the applicant UI. This is missing a specified text size, so that
+   * should be added by other button style constants that use this as a base.
+   */
+  private static final String BUTTON_BASE =
+      StyleUtils.joinStyles(
+          "block", "py-2", "text-center", "rounded-full", "border", "border-transparent");
+
+  /** Base styles for buttons with a solid background color. */
+  private static final String BUTTON_BASE_SOLID =
+      StyleUtils.joinStyles(
+          BUTTON_BASE,
+          BaseStyles.BG_SEATTLE_BLUE,
+          "text-white",
+          "rounded-full",
+          StyleUtils.hover("bg-blue-700"),
+          StyleUtils.disabled("bg-gray-200", "text-gray-400"));
+
+  /** Base styles for semibold buttons with a solid background. */
+  private static final String BUTTON_BASE_SOLID_SEMIBOLD =
+      StyleUtils.joinStyles(BUTTON_BASE_SOLID, "font-semibold", "px-8");
```

Comment:
**optional:** BUTTON_BASE_SOLID is only used in BUTTON_BASE_SOLID_SEMIBOLD, so you could consider combining those unless you think there will be a use case for a non-semibold version of it in the future.

Same for BUTTON_BASE_OUTLINE and BUTTON_BASE_OUTLINE_SEMIBOLD

Reply:
Good catch - consolidated.